NineChronicles.Headless.Executable/Commands/ReplayCommand.cs

```diff
@@ -40,29 +41,21 @@ public void Help([FromService] ICoconaHelpMessageBuilder helpMessageBuilder)
 
         [Command(Description = "Evaluate tx and calculate result state")]
         public int Tx(
-            [Argument("TX-PATH", Description = "A JSON file path of tx.")]
+            [Option('t', Description = "A JSON file path of tx.")]
```

Comment:
We can provide both short and long version of option name. Do you have any idea about this?

Reply:
The `Cocona.Command` attribute provide automatically short and long versions of option name.👍

```text
$ dotnet run -- replay blocks -h
>
Usage: NineChronicles. replay blocks [--store-path <String>] [--start-index <Int64>] [--end-index <Int64>] [--repeat-count <Int32>] [--verbose] [--output-path <String>] [--help]

Evaluate blocks and check state root hash

Options:
  -s, --store-path <String>     An absolute path of block storage.(rocksdb) (Required)
  -i, --start-index <Int64>     Target start block height. Tip as default. (Min: 1) (Default: 1)
  -e, --end-index <Int64>       Target end block height. Tip as default. (Min: 1)If not set, same as argument "-i".
  -r, --repeat-count <Int32>    Repeat count. (Min: 1)If not set, default is 1. (Default: 1)
  -v, --verbose                 Verbose mode.
  -o, --output-path <String>    The path of output file.
  -h, --help                    Show help message
```